lib/resty/upstream/healthcheck.lua

```diff
@@ -701,4 +701,57 @@ function _M.status_page()
     return concat(bits)
 end
 
+function _M.status_page_one()
```

Comment:
I'm not happy with the function name. Maybe we should reuse the existing `status_page` function and just make it accept a function parameter? It's more flexible than hard-coding the "upstream" URI arg name in the implementation I think.